From c064211d94884f2c131c5496e60f591511c5462d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Thu, 28 Aug 2014 15:02:12 +0200
Subject: wined3d: Remove the texture destroy glFinish.

This should work now that surface_decref is essentially gone and the
container is never unset.
---
 dlls/wined3d/texture.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/dlls/wined3d/texture.c b/dlls/wined3d/texture.c
index 09ac38f..6f78292 100644
--- a/dlls/wined3d/texture.c
+++ b/dlls/wined3d/texture.c
@@ -146,13 +146,6 @@ static void wined3d_texture_cleanup(struct wined3d_texture *texture)
 
     TRACE("texture %p.\n", texture);
 
-    /* Because sub_resource_cleanup interferes with GL resources */
-    if (wined3d_settings.cs_multithreaded)
-    {
-        FIXME("Waiting for cs.\n");
-        device->cs->ops->finish(device->cs);
-    }
-
     for (i = 0; i < sub_count; ++i)
     {
         struct wined3d_resource *sub_resource = texture->sub_resources[i];
-- 
2.1.3

